superset/assets/src/dashboard/containers/DashboardHeader.jsx

```diff
@@ -69,14 +69,16 @@ function mapStateToProps({
     ).text,
     expandedSlices: dashboardState.expandedSlices,
     css: dashboardState.css,
+    colorNamespace: undoableLayout.present[COLOR_SCHEME_ID].meta.colorNamespace,
```

Comment:
for line `layout: undoableLayout.present,` can you remove `COLOR_SCHEME_ID` and `DASHBOARD_HEADER_ID` key from `layout` prop? `layout` prop will be the content to be persisted into dashboard table as `position_json`. COLOR_SCHEME_ID section should not being saved as part of position_json.

Reply:
Thanks for catching that. How about I move the color scheme info to dashboardState? Changing colors will not be an undoable/redoable action if I move it to dashboardState but since the dashboard re-renders and reflects the change immediately, I think it is fine that it is not undoable/redoable.